packages/fonts/src/utils/index.js

```diff
@@ -16,3 +16,4 @@
 
 export { default as getFontCSS } from './getFontCSS';
 export { default as getGoogleFontURL } from './getGoogleFontURL';
+export { default as getFontDataFromUrl } from './getFontDataFromUrl';
```

Comment:
Was it not exported purposely because it was increasing the bundle size?  🤔

Reply:
It shouldn't be in the fonts package in the first place. It is only used in `wp-dashboard`.